### PR TITLE
Homeworkbranch

### PR DESCRIPTION
**Pull request by KristineMarkarova**

### CODE REVIEW

**Comment by KristineMarkarova** [comment]
SEE THIS ONE     ad2ae90
gliffy-example

File: src/main/java/work/work.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        System.out.print("Please enter the second number: ");
+        int newnum2 = numeral.nextInt();
+        System.out.print("Please enter the tihrd number: ");
+        int newnum3 = numeral.nextInt();
+
+        if (newnum1 > newnum2 && newnum1 > newnum3)
+            System.out.println("The greatest number is: " + newnum1);
+        else if (newnum2 > newnum1 && newnum2 > newnum3)
+            System.out.println("The greatest number is: " + newnum2);
+        else
```
Comment (HamoAtayan) [inline src/main/java/work/work.java:33]: not covering equal case

File: src/main/java/work/work.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            System.out.println("The greatest number is: " + newnum3);
+//4. Write a Java program to get a number from the user and print whether it is positive or negative
+        Scanner x = new Scanner(System.in);
+        System.out.print("Please enter your number: ");
+        int xx = x.nextInt();
+
+        if (xx > 0)
+            System.out.println("Your number is positive: " + xx);
+        else
+            System.out.println("Your number is negative: " + xx);
```
Comment (HamoAtayan) [inline src/main/java/work/work.java:43]: not covered 0 case

**Review by HamoAtayan — APPROVED** [review]

**Comment by KristineMarkarova** [comment]
do not need this

**Comment by KristineMarkarova** [comment]
c

**Pull request closed without merging**